doc/telestrat.sgml

```diff
@@ -0,0 +1,177 @@
+<!doctype linuxdoc system>
+
+<article>
+
+<title>Oric Telestrat-specific information for cc65
+<author>
+<url url="mailto:jede@oric.org" name="Jede">,<newline>
+
+<date>2017-01-22
+
+<abstract>
+An overview over the Telestrat (telemon 3.0 : http://orix.oric.org) runtime system as it is implemented for the cc65 C
+compiler.
+</abstract>
+
+<!-- Table of contents -->
+<toc>
+
+<!-- Begin the document -->
+
+<sect>Overview<p>
+
+This file contains an overview of the Telestrat runtime system as it comes with the
+cc65 C compiler. It describes the memory layout, Telemon3.0-specific header files,
+available drivers, and any pitfalls specific to that platform.
+
+Please note that Telemon3.0-specific functions are just mentioned here, they are
+described in detail in the separate <url url="funcref.html" name="function
+reference">. Even functions marked as "platform dependent" may be available on
+more than one platform. Please see the function reference for more
+information.
+
+
+
+<sect>Binary format<p>
+
+The standard binary output format generated by the linker for the Telemon 3.0 target
+is a machine language program with a 20 bytes header described here : http://orix.oric.org/doku.php?id=orix:header
+
+
+
+<sect>Memory layout<p>
+
+In the standard setup, cc65-generated programs use the memory from
+&dollar;0801 to &dollar;9800; so, nearly 37K of memory (including the stack) is
+available. ROM calls are possible without further precautions.
+
+
+
+Special locations:
+
+<descrip>
+  <tag/Stack/
+  The C runtime stack is located at &dollar;97FF (or &dollar;B3FF), and grows
+  downwards.
+
+  <tag/Heap/
+  The C heap is located at the end of the program, and grows towards the C
+  runtime stack.
+
+</descrip><p>
+
+
+
+<sect>Platform-specific header files<p>
+
+Programs containing Telemon 3.0 -specific code may use the <tt/telemon.h/ header file.
+
+
+<sect1>Telemon 3.0-specific functions<p>
+
+The functions listed below are special for the Telemon 3.0. See the <url
+url="funcref.html" name="function reference"> for declaration and usage.
+
+<itemize>
+<item>explode
+<item>ping
+<item>shoot
+<item>zap
+<item>oupsx
```

Comment:
The main problem for : 

atmos.h would include oric.h
telestrat.h would include oric.h

there are atmos_zap(), atmos_explode(). Theses functions are tje same for telestrat. I would like to have zap(), ping() instead. But, in this way, some code will be broken.

Let me know, because i don't know how to do this fix